ratis-grpc/src/main/java/org/apache/ratis/grpc/server/GrpcService.java

```diff
@@ -111,13 +113,21 @@ private GrpcService(RaftServer raftServer, Supplier<RaftPeerId> idSupplier, int
 
     this.clientProtocolService = new GrpcClientProtocolService(idSupplier, raftServer);
 
+    MetricServerInterceptor monitoringInterceptor = new MetricServerInterceptor(
```

Comment:
Better identifier for the Server proxy.

Reply:
i think we can add the client id and raftserverImpl id...

Reply:
that is also created after the server is started.

Reply:
fixed.